packages/flutter/test/widgets/text_golden_test.dart

```diff
@@ -59,4 +59,72 @@ void main() {
       skip: !Platform.isLinux,
     );
   });
+
+
+  testWidgets('Text Foreground', (WidgetTester tester) async {
+    const Color black = const Color(0xFF000000);
+    const Color red = const Color(0xFFFF0000);
+    const Color blue = const Color(0xFF0000FF);
+    final Shader linearGradient = const LinearGradient(colors: <Color>[red, blue]).createShader(new Rect.fromLTWH(0.0, 0.0, 50.0, 20.0));
+
+    await tester.pumpWidget(
+      new RepaintBoundary(
```

Comment:
If you put the RepaintBoundary in a Center widget, the image will be much more reasonably sized.